src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
+  const carCost = 40;
+  const rent = carCost * days;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
There is a logical error in the discount calculation. If the number of days is 7 or more, the discount should be 50, not 20. Currently, the code sets the discount to 20 if days are 3 or more, and then to 50 if days are 7 or more, which is correct. However, the logic should ensure that the discount for 7 or more days overrides the discount for 3 or more days. Consider using an `else if` statement for the 3-day discount to prevent both discounts from being applied.